### ISSUE TITLE
Using a local file - just put "file:///home/$user/htmlfile.html

### ISSUE THREAD

**Issue by andiohn** [debloper/piosk#22]
Just thought I'd highlight that you can run local content.

**Comment by debloper** [comment]
Yeah, absolutely... any URL that you can open in browser should work.

How about we add it as an example URL to the sample config file which loads up on first setup, to give users a hint that they can use local `file://` protocol or `localhost:port` server etc?

**Comment by andiohn** [comment]
Absolutely! That would exactly work it. 


I'll share a time and weather locally run website that you can use!





-John



---- On Fri, 19 Jul 2024 16:42:55 -0600 ***@***.*** wrote ----



Yeah, absolutely... any URL that you can open in browser should work.

How about we add it as an example URL to the sample config file which loads up on first setup, to give users a hint that they can use local file:// protocol or localhost:port server etc?
